tests/bud.bats

```diff
@@ -1217,6 +1221,7 @@ function _test_http() {
   target=url-image
   url=https://raw.githubusercontent.com/containers/buildah/main/tests/bud/from-scratch/Dockerfile.bogus
   run_buildah 125 build --signature-policy ${TESTSDIR}/policy.json -t ${target} ${url}
+  expect_output "no FROM statement found"
```

Comment:
That's correct, but not obvious.  The issue is the URL set on line 1222 is invalid; that file does not exist there.  I'm open to a better test name here, and/or changing the value sent into the URL field to be more obvious.

Reply:
i think test could be renamed to `Invalid dockerfile from URL`, cause URL seems to be valid its just we get a 404 instead of a valid dockerfile, so buildah tries to process `404` which is invalid dockerfile in the end.

Reply:
I feel rather strongly that buildah should be more clear on the error: "Could not fetch <URL>: 404" would be more helpful to an end user than "oh, I think I fetched your URL but couldn't find a FROM line". And I agree that "Invalid URL" is a misnomer. But neither can be solved in this PR.